stream-chat-android-ui-components-sample/build.gradle

```diff
@@ -161,10 +161,10 @@ dependencies {
     implementation Dependencies.coilGif
     implementation Dependencies.shimmer
 
-    debugImplementation Dependencies.flipper
-    debugImplementation Dependencies.flipperNetwork
-    debugImplementation Dependencies.flipperLoader
-    debugImplementation Dependencies.fragmentTest
+//    debugImplementation Dependencies.flipper
+//    debugImplementation Dependencies.flipperNetwork
+//    debugImplementation Dependencies.flipperLoader
+//    debugImplementation Dependencies.fragmentTest
```

Comment:
I tested and Flipper works fine. The problem is caused by `Dependencies.fragmentTest`

Reply:
Oh, nice. Let me test this better then.